### PR TITLE
removed duplicated annotation in rasa-x-ingress.yaml

### PR DESCRIPTION
**Pull request by dongsu**
The removed block in this PR exists already in the `rasa-x-ingress.yaml` template:
https://github.com/RasaHQ/rasa-x-helm/blob/d86da0fdb7bf64deb85acd305314521dae1203dd/charts/rasa-x/templates/rasa-x-ingress.yaml#L15-L17

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @Tawakalt will take a look at it as soon as possible ✨

**Comment by tczekajlo** [comment]
@dongsu Welcome to the rasa-x-helm project 🎉  Thanks a lot for your first contribution. LGTM, the only thing left to do is bump the chart version to 1.7.14.

You can do this by updating the value in the `Chart.yaml` file 

https://github.com/RasaHQ/rasa-x-helm/blob/master/charts/rasa-x/Chart.yaml#L4

File: charts/rasa-x/templates/ingress.yaml [hunk 1]
```diff
@@ -49,12 +49,28 @@ spec:
...
+          - path: {{ . }}
+            pathType: Prefix
+            backend:
+              service:
+                name: {{ $fullName }}-nginx
+                port:
+                  number: {{ $svcPort }}
+          {{- end }}
+    {{- end }}
+{{- else -}}
```
Comment (b-quachtran) [inline charts/rasa-x/templates/ingress.yaml:62]:
```suggestion
{{- end }}
```

**Comment by dongsu** [comment]
it seems like the change is in. closing this PR

**Pull request closed without merging**